### PR TITLE
Update dp-renderer to 1.8.0 and make dp-design-system the priority over sixteens

### PR DESCRIPTION
**Pull request by rav-pradhan**
### What

Updated `dp-renderer` to 1.8.0 and updated mapper functions to use `SixteensVersion` where pertinent

### How to review

- Sense check code
- Check pages handled by the dataset controller locally to ensure that they are loading in either Sixteens or dp-design-system. If it is a CMD or legacy dataset page, then that should be loading in Sixteens. If it's the Census dataset landing page, then dp-design-system should be loaded in

### Who can review

Anyone

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -34,9 +34,9 @@ func Get() (*Config, error) {
 	}
 
 	if cfg.Debug {
-		cfg.PatternLibraryAssetsPath = "http://localhost:9000/dist"
+		cfg.PatternLibraryAssetsPath = "http://localhost:9001/dist/assets"
```
Comment (mr-nick17) [inline config/config.go:37]: Incorrect local port number
```suggestion
		cfg.PatternLibraryAssetsPath = "http://localhost:9002/dist/assets"
```

File: mapper/mapper.go [hunk 2]
```diff
@@ -29,9 +29,10 @@ type TimeSlice []time.Time
 
 // Dimension names
 const (
-	DimensionTime      = "time"
-	DimensionAge       = "age"
-	DimensionGeography = "geography"
+	DimensionTime          = "time"
+	DimensionAge           = "age"
+	DimensionGeography     = "geography"
+	CurrentSixteensVersion = "67f6982"
```
Comment (mr-nick17) [inline mapper/mapper.go:35]: I think `SixteensVersion` would have been a better name. Have the name prefixed with `Current` implies that it's the most up to date which may not be the case

**Review by mr-nick17 — COMMENTED** [review]
Not solely related to this service but there are a number of console.errors which I believe are related to code in the `dp-design-system` 
image
I would sumize that this [method](https://github.com/ONSdigital/dp-design-system/blob/ba32e79996fe8874353663b68da5d5311ccf3427/gulpfile.js#L61) needs a closer look

**Comment by rav-pradhan** [comment]
> Not solely related to this service but there are a number of console.errors which I believe are related to code in the `dp-design-system` image I would sumize that this [method](https://github.com/ONSdigital/dp-design-system/blob/ba32e79996fe8874353663b68da5d5311ccf3427/gulpfile.js#L61) needs a closer look

As discussed, this was a bug in the ONS DS that has been fixed by bumping the npm module version. I will update this PR with the new commit hash once dp-design-system is updated

**Review by mr-nick17 — DISMISSED** [review]
LGTM

**Review by mr-nick17 — APPROVED** [review]
LGTM 👍

**Pull request merged**